src/Integration.Vsix.UnitTests/Suppression/DelegateInjectorTests.cs

```diff
@@ -38,7 +39,7 @@
 namespace SonarLint.VisualStudio.Integration.UnitTests.Suppression
 {
     [TestClass]
-    public class DelegateInjectorTests
+    public class DelegateInjectorTests : MarshalByRefObject
```

Comment:
@valhristov asked whether the tests would interfere with each other if run in parallel since they all use the same app domain. I tried it from the command line and they ran ok. However, there is a problem running the tests multiple times in VS, so I've changed the tests so each one is run in a separate app domain.

This class is now _MarshalByRef_ so it can be created in the remote app domain.